src/components/WelcomeModal/WelcomeModal.stories.tsx

```diff
@@ -37,8 +37,8 @@ export const MovieCard = () => (
   <MockedProvider>
     <CarouselCard
       card={{
-        title: "A Title",
-        subtitle: "A Subtitle",
+        title: "Components/A Title",
+        subtitle: "Components/A Subtitle",
```

Comment:
Same question about `Components/` in title

<img width="544" alt="Screen Shot 2022-06-29 at 1 21 24 PM" src="https://user-images.githubusercontent.com/47064971/176497871-975a9bde-0548-4844-9b79-05b5b31873fe.png">